src/scripts/__tests__/__snapshots__/fetchStatsFromGA.js.snap

```diff
@@ -0,0 +1,1183 @@
+  Object {
+    "date": "2020-07-15",
+    "docId": "testID3",
+    "docUserId": undefined,
+    "fetchedAt": 2020-01-01T00:00:00.000Z,
+    "stats": Object {
+      "webUser": 2,
+      "webVisit": 5,
+    },
+    "type": "article",
+  },
+  "article_testID4_2020-07-15",
+  Object {
+    "date": "2020-07-15",
+    "docId": "testID4",
+    "docUserId": undefined,
+    "fetchedAt": 2020-01-01T00:00:00.000Z,
+    "stats": Object {
+      "webUser": 25,
+      "webVisit": 69,
+    },
+    "type": "article",
+  },
+  "article_testID5_2020-07-15",
+  Object {
+    "date": "2020-07-15",
+    "docId": "testID5",
+    "docUserId": undefined,
+    "fetchedAt": 2020-01-01T00:00:00.000Z,
+    "stats": Object {
+      "webUser": 46,
+      "webVisit": 109,
+    },
+    "type": "article",
+  },
+]
+`;
+
+exports[`fetchStatsFromGA processReport with stubbed upsertDocStats should aggregate rows of data cross pages 3`] = `
+Object {
+  "date": "2020-07-16",
+  "docId": "testID5",
+  "docType": "article",
+  "docUserId": undefined,
+  "sourceType": "WEB",
+  "stats": Object {
+    "webUser": 4,
+    "webVisit": 10,
+  },
+}
+`;
+
+exports[`fetchStatsFromGA processReport with stubbed upsertDocStats should aggregate rows of data cross pages 4`] = `
+Array [
+  "article_testID5_2020-07-15",
+  Object {
+    "date": "2020-07-15",
+    "docId": "testID5",
+    "docUserId": undefined,
+    "fetchedAt": 2020-01-01T00:00:00.000Z,
+    "stats": Object {
+      "webUser": 49,
+      "webVisit": 115,
+    },
+    "type": "article",
+  },
+  "article_testID1_2020-07-16",
+  Object {
+    "date": "2020-07-16",
+    "docId": "testID1",
+    "docUserId": undefined,
+    "fetchedAt": 2020-01-01T00:00:00.000Z,
+    "stats": Object {
+      "webUser": 1,
+      "webVisit": 2,
+    },
+    "type": "article",
+  },
+  "article_testID2_2020-07-16",
+  Object {
+    "date": "2020-07-16",
+    "docId": "testID2",
+    "docUserId": undefined,
+    "fetchedAt": 2020-01-01T00:00:00.000Z,
+    "stats": Object {
+      "webUser": 1,
+      "webVisit": 2,
+    },
+    "type": "article",
+  },
+  "article_testID5_2020-07-16",
+  Object {
+    "date": "2020-07-16",
+    "docId": "testID5",
+    "docUserId": undefined,
+    "fetchedAt": 2020-01-01T00:00:00.000Z,
+    "stats": Object {
+      "webUser": 4,
+      "webVisit": 10,
+    },
+    "type": "article",
+  },
+]
+`;
+
+exports[`fetchStatsFromGA processReport with stubbed upsertDocStats should call bulkUpdates with right params 1`] = `
+Array [
+  Object {
+    "update": Object {
+      "_id": "article_testID1_2020-07-15",
+      "_index": "analytics",
+      "_type": "doc",
+    },
+  },
+  Object {
+    "script": Object {
+      "params": Object {
+        "date": "2020-07-15",
+        "docId": "testID1",
+        "docUserId": undefined,
+        "fetchedAt": 2020-01-01T00:00:00.000Z,
+        "stats": Object {
+          "webUser": 1,
+          "webVisit": 2,
+        },
+        "type": "article",
+      },
+      "source": "
```

Comment:
If we are going to repeat the `source` of the script many many times in a batch, I think we may consider using stored scripts.
https://www.elastic.co/guide/en/elasticsearch/reference/6.8/modules-scripting-using.html#modules-scripting-stored-scripts

We can update the stored script everytime the cron job starts to make sure the script stored in database is the latest one.